releasePipeline/50-tag-github-repositories.sh

```diff
@@ -0,0 +1,167 @@
+}
+error() { printf "${red}✖ %s${reset}\n" "$@"
+}
+warn() { printf "${tan}➜ %s${reset}\n" "$@"
+}
+bold() { printf "${bold}%s${reset}\n" "$@"
+}
+note() { printf "\n${underline}${bold}${blue}Note:${reset} ${blue}%s${reset}\n" "$@"
+}
+
+
+#-----------------------------------------------------------------------------------------                   
+# Main logic.
+#-----------------------------------------------------------------------------------------   
+
+mkdir -p ${WORKSPACE_DIR}/temp
+
+
+function get_galasa_version_to_be_released {
+    h1 "Working out the version of Galasa to test and release."
+
+    url="https://development.galasa.dev/main/maven-repo/obr/dev/galasa/dev.galasa.uber.obr/"
+    curl $url > temp/galasa-version.txt -s
+    rc=$?; 
+    if [[ "${rc}" != "0" ]]; then 
+      error "Failed to get galasa version"
+      exit 1
+    fi
+
+    # Note: We take the 2nd line which has an "<a href" string on... hopefully it won't change...
+    galasa_version=$(cat temp/galasa-version.txt | grep "<a href" | head -2 | tail -1 | cut -f2 -d'"' | cut -f1 -d'/')
+
+    success "Galasa version to be tested and released is ${galasa_version}"
+    export galasa_version
+}
+
+
+function set_kubernetes_context {
+    h1 "Setting the kubernetes context to be cicsk8s, using namespace galasa-build"
+    kubectl config set-context cicsk8s --namespace=galasa-build
+    rc=$?
+    if [[ "${rc}" != "0" ]]; then 
+        error "Failed. rc=${rc}"
+        exit 1
+    fi
+    
+}
+
+function tag_galasa_github_repositories {
+
+    h1 "Tagging the github repositories..."
+
+    cd ${WORKSPACE_DIR}/temp
+    yaml_file=${WORKSPACE_DIR}/temp/tag-galasa-repositories.yaml
+    cat << EOF > $yaml_file
+#
+# Copyright contributors to the Galasa project 
+#
+kind: PipelineRun
+apiVersion: tekton.dev/v1beta1
+metadata:
+  generateName: tag-galasa-
+  annotations:
+    argocd.argoproj.io/compare-options: IgnoreExtraneous
+    argocd.argoproj.io/sync-options: Prune=false
+#
+#
+#
+spec:
+#
+#
+#
+  pipelineRef:
+    name: branch-tag-galasa
+  
+  serviceAccountName: galasa-build-bot
+
+  podTemplate:
+    volumes:
+    - name: githubcreds
+      secret:
+        secretName: github-token
+    - name: harborcreds
+      secret:
+        secretName: harbor-creds-yaml
+
+  params:
+  - name: distBranch
+    value: "release"
+#
+#  Tag must be in the format v0.0.0
+#
+  - name: tag
+    value: "v${galasa_version}"
+EOF
+
+    cmd="kubectl -n galasa-build create -f $yaml_file"
+    info "Command is $cmd"
+    $cmd
+    rc=$?
+    if [[ "${rc}" != "0" ]]; then
+        error "Failed to create the resources image. rc=$rc"
+        exit 1
+    fi
+
+    success "Resources image build pipeline kicked off OK."
+}
+
+set_kubernetes_context
+get_galasa_version_to_be_released
+
+tag_galasa_github_repositories
+
+note "Now wait for the 'tag-galasa-*' pipeline to complete."
+note "Expect it to take about a minute."
+note "Check that it passed"
+
```

Comment:
same here we can check for the status of the pipelineRun if is completed or not?

Reply:
same response. We can, but not in this PR.